src/client/lib/ThrottledBreachedPasswordCheck.ts

```diff
@@ -0,0 +1,73 @@
+import { PasswordValidationResult } from '@/shared/lib/PasswordValidation';
+
+export class ThrottledBreachedPasswordCheck {
+  checkInterval = 500; //ms
```

Comment:
throttle requests to breached password endpoint to twice per second, timing out after 1 second